client/blocks/signup-form/crowdsignal.scss

```diff
@@ -44,10 +44,8 @@ body.is-section-signup .layout .signup.is-crowdsignal .step-wrapper {
 	}
 }
 
-.signup-form__crowdsignal {
+.signup.is-crowdsignal {
```

Comment:
Whereabouts are these classnames output? I must be missing something, can't find them in the markup 😅

Reply:
Yeah not easy to find, [here](https://github.com/Automattic/wp-calypso/blob/3212bd8f1dbdfc4fed931450ad2d0494ebd1685f/client/signup/main.jsx#L866)